### ISSUE TITLE
Pokemon Soulsilver freeze issue

### ISSUE THREAD

**Issue by SyedDawoud** [TuxSH/TWLSaveTool#18]
TWLSaveTool can't read the Pokemon soulsilver save. It gets stuck on start and it starts only when soulsilver cartridge is pulled out. I am using the Latest version.

**Comment by brunoviek** [comment]
Same problem! Any other tool?

**Comment by SyedDawoud** [comment]
No. I haven't had any success backing up the save by any other tool
